include/pfs/parsers.hpp

```diff
@@ -183,9 +183,9 @@ class file_parser
         return output;
     }
 
-protected:
     using value_parser =
```

Comment:
This member should not be used from a class that doesn't inherit from `file_parser`.
So I guess we should find a different way to export that to your generic `value_parser` implementation.